src/components/ChatLog.test.js

```diff
@@ -1,49 +1,49 @@
-import React from "react";
-import "@testing-library/jest-dom/extend-expect";
-import ChatLog from "./ChatLog";
-import { render, screen } from "@testing-library/react";
+import React from 'react';
```

Comment:
Good catch on changing the single quotes! ESLinters are really finicky about style rules!